lib/scan_code/scan.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:flutter/material.dart';
+import 'package:barcode_scan2/barcode_scan2.dart';
+import 'package:flutter/services.dart';
+import 'package:generator/widgets/appbar.dart';
+
+class ScanPage extends StatefulWidget {
+  @override
+  _ScanPageState createState() => _ScanPageState();
+}
+
+class _ScanPageState extends State<ScanPage> {
+  @override
+  void initState() {
+    super.initState();
+    _scan();
+  }
+
+  String qrCodeResult = "";
```

Comment:
**suggestion:** use a state management approach to control the state of widgets like these

Reply:
Thanks, will start do that from now.